pkg/stanza/fileconsumer/internal/tracker/tracker.go

```diff
@@ -52,13 +55,18 @@ type fileTracker struct {
 	archiveIndex   int
 }
 
+var errInvalidValue = errors.New("invalid value")
+
+var archiveIndexKey = "knonwFiles_ai"
```

Comment:
Let's be verbose so the valu is self explanatory

```suggestion
var archiveIndexKey = "knownFilesArchiveIndex"
```